cobbler/api.py

```diff
@@ -2026,3 +2026,53 @@ def mkloaders(self):
         """
         action = mkloaders.MkLoaders(self)
         action.run()
+
+    # ==========================================================================
+
+    def input_string_or_list_no_inherit(
+        self, options: Optional[Union[str, list]]
+    ) -> list:
+        """
+        .. seealso:: :func:`~cobbler.utils.input_converters.input_string_or_list_no_inherit`
```

Comment:
I would kindly ask that you adjust the docstrings of the functions. This would link to the wrong documentation.